### DISCUSSION TITLE
⭐程序更新/问题修复公告

### DISCUSSION THREAD

**Discussion by Muhu-C** [Muhu-C/DateTimer#3] [公告]
# 1.0.4 已更新！  
+ **在 Windows 7 设备的测试中发现 .NET Framework 4.8.1 无法兼容，但是 .NET Framework 4.8.0 可以完全兼容，且此前版本没有 Segoe Fluent Icons 的提示，故推出 1.0.4 版本！**  
+ **更新时间：8月10日**  
+ **经测试，本程序兼容运行时版本为 .NET Framework 4.8(兼容Win7) 和 .NET Framework 4.8.1(Win10及以上专用)**

**Comment by Muhu-C** [comment]
### OK 啊已经把程序未知出错弹窗加上了

**Comment by MC118CN** [comment]
# 1.0.4.1 更新！
## 添加功能：防止多次开启程序提示
